### PR TITLE
Fixes #92 - Add HTML and CSS for schedule

### LINKED ISSUE #92: HTML/CSS - Schedule - est. 1.5 day

### PR DESCRIPTION
**Pull request by Ctannerweb**
Still need two illustrations and speaker content

### CODE REVIEW

**Comment by Ctannerweb** [comment]
Will add ellipsis to first paragraph when we have speaker content and if we need it

File: maine/schedule.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <html lang="en-US">
   <head>
     <meta charset="UTF-8">
-    <title>Schedule | UX Camp</title>
+    <title>UX Camp</title>
```
Comment (acacheung) [inline maine/schedule.html]: Schedule | UX Camp is probs better

File: maine/schedule.html [hunk 2]
```diff
@@ -16,8 +16,190 @@
...
+        </symbol>
+<symbol id="watch" viewBox="0 0 571 1024">
+<title>watch</title>
+<path class="path1" d="M570.785 512c0-102.637-51.674-192.948-135.585-243.674v-221.393c0-25.6-17.778-46.933-44.8-46.933h-207.881c-26.785 0-49.541 21.333-49.541 46.933v223.052c-83.911 50.963-132.978 140.326-132.978 242.015s49.067 191.052 132.978 242.015v225.659c0 25.6 22.756 44.326 49.541 44.326h207.881c26.785 0 44.8-18.726 44.8-44.326v-224c83.911-50.726 135.585-140.8 135.585-243.674zM166.637 46.933c0-7.111 7.585-13.511 16.119-13.511h207.881c8.296 0 11.141 6.4 11.141 13.511v203.378c-33.659-16.356-74.193-25.363-116.148-25.363-42.904 0-85.333 9.481-118.756 26.548v-204.563h-0.237zM401.541 979.674c0 7.111-2.844 10.667-11.141 10.667h-207.881c-8.296 0-16.119-3.556-16.119-10.667v-207.17c33.659 17.067 75.852 26.548 118.756 26.548 41.956 0 82.489-9.244 116.148-25.363l0.237 205.985zM287.763 765.393c-139.852 0-253.393-113.778-253.393-253.393s113.778-253.393 253.393-253.393 253.63 113.778 253.63 253.393c0 139.852-113.778 253.393-253.63 253.393z"></path>
+<path class="path2" d="M316.207 539.733h-117.57v-33.659h100.741v-168.059h33.659v184.889c0 9.244-7.585 16.83-16.83 16.83z"></path>
+</symbol>
```
Comment (acacheung) [inline maine/schedule.html]: indentation

File: maine/schedule.html [hunk 3]
```diff
@@ -16,8 +16,189 @@
...
+        <span class="t-hidden">Maine</span>
+      </a>
+      <a class="club-sandwich" href="#">
+        <span class="club-sandwich__top"></span>
+        <span class="club-sandwich__middle"></span>
+        <span class="club-sandwich__bottom"></span>
+        <span class="t-hidden">Main navigation</span>
+      </a>
+      <nav class="nav__links">
+        <a class="nav__link" href="schedule.html">Schedule</a>
```
Comment (acacheung) [inline maine/schedule.html]: active

File: maine/schedule.html [hunk 4]
```diff
@@ -16,8 +16,189 @@
...
+        <svg class="heading__icon"><use xlink:href="#watch"></use></svg>
+        <h1 class="heading">Schedule</h1>
+        <h2 class="subheading">july 18–20, 2015</h2>
+      </div>
+      <div class="l-schedule">
+        <div class="day">
+          <h2 class="day__name">Friday</h2>
+          <div class="day__time-slot">
+            <time class="time-slot__time">12:00-2:00 pm</time>
+            <h3 class="time-slot__activity">Meet fellow attendees & settle in</h3>
```
Comment (acacheung) [inline maine/schedule.html]: &amp;

File: maine/schedule.html [hunk 5]
```diff
@@ -16,8 +16,189 @@
...
+          <div class="day__time-slot">
+            <time class="time-slot__time">12:00-1:00 pm</time>
+            <h3 class="time-slot__activity">Lunch</h3>
+          </div>
+          <div class="day__time-slot">
+            <time class="time-slot__time">1:00-4:00 pm</time>
+            <h3 class="time-slot__event">Prototyping in the Woods</h3>
+            <div class="t-link__wrap">
+              <a class="t-link spot-color">Roz Duffy</a>
+              <span> &amp; </span>
```
Comment (acacheung) [inline maine/schedule.html]: the spaces before and after the ampersand don't do anything

File: maine/schedule.html [hunk 6]
```diff
@@ -16,8 +16,189 @@
...
+      </div>
+    </main>
+    <footer class="footer">
+      <div class="l-footer">
+        <div class="footer--left">
+          <h2 class="footer__heading">UX Camp 2016</h2>
+          <ul class="footer__contacts">
+            <li class="footer__contact">
+              <span class="footer__contact__key">Email:</span>
+              <a class="footer__contact__value spot-color" href="mailto:UXCamp@gmail.com">UXCamp@gmail.com</a>
```
Comment (acacheung) [inline maine/schedule.html]: should this be steve's email?

File: maine/schedule.html [hunk 7]
```diff
@@ -16,8 +16,189 @@
...
+            </li>
+            <li class="footer__contact">
+              <span class="footer__contact__key">Phone:</span>
+              <a class="footer__contact__value spot-color" href="tel:123-456-7890">(123)456-7890</a>
```
Comment (acacheung) [inline maine/schedule.html]: steve's phone number?

File: maine/schedule.html [hunk 8]
```diff
@@ -16,8 +16,189 @@
...
+          </div>
+          <div class="footer__links">
+            <h3 class="footer__links__heading">Locations</h3>
+            <a class="footer__link spot-color" href="staff.html">California</a>
+            <a class="footer__link spot-color" href="archive.html">Colorado</a>
+            <a class="footer__link spot-color" href="code.html">Maine</a>
+          </div>
+        </div>
+        <span class="footer__dockyard">
+          A production from
```
Comment (acacheung) [inline maine/schedule.html]: Built by

**Pull request merged**